content/blog/2025-05-08-go-client-GA.md

```diff
@@ -0,0 +1,501 @@
++++
+title= "Valkey GLIDE Go Client: Now Generally Available!"
+description = "Valkey GLIDE Go client reaches general availability. Read to learn more about the go client designed for performance and developer productivity"
+date= 2025-05-08 01:01:01
+authors= [ "niharikabhavaraju", "jbrinkman"]
++++
+
+The Valkey GLIDE contributors are pleased to announce the general availability (GA) release of the GLIDE (General Language Independent Driver for the Enterprise) Go client. This release brings the power and reliability of Valkey to Go developers with an API designed for performance and developer productivity.
+
+Vakey GLIDE 2.0 brings exciting new features for all of our language clients (Java, Node.js, Python, and now Go). These features include pipelines, OpenTelemetry support, MacOS x86 support, and improved performance and bug fixes.
+
+The Go client extends Valkey GLIDE to the Go community, offering a robust client that's built on the battle-tested Rust core. This client library is a thoughtfully designed experience for Go developers who need reliable, high-performance data access.
+
+## What's new in GA?
+
+- The major addition in the GA release is comprehensive [batch support](#batch-support) that includes both transactions and pipelines.
+- Complete support for all Valkey commands encompassing scripting, functions, pubsub, server management, and all other operations in both standalone and cluster modes.
+- Support for OpenTelemetry tracing which has been added to all language clients, including Go.
```

Comment:
Should we highlight context support?
And revised structs in responses

Reply:
I added context. The revised structs I think are a little too low level, like refactoring package names and moving internal implementation details.